draft.xml

```diff
@@ -191,6 +191,144 @@ Content-Type: application/json
       </section>
     </section>
     
+    <section title="Authenticator Association List Endpoint">
+      <t>
+        The authenticator list endpoint is an OAuth 2.0 endpoint defined in
+        this document that is designed to allow listing all the authenticators
+        associated with a resource owner.  The authenticator list endpoint
+        MUST accept HTTP GET messages.  The authenticator
+        list endpoint MUST be protected by a transport-layer security
+        mechanism.
+      </t>
+      <t>
+        The authenticator associate endpoint MUST be an OAuth 2.0 protected
+        resource and it MUST accept access tokens as defined by <xref target='RFC6750'/>
+        to limit listing of authenticators to only previously authorized parties.  The method
+        by which the access token is obtained by the client is generally out of
+        band; one of such mechanisms is defined by this specification.  The method
+        by which the access token is verified and validated by the authenticator
+        associate endpoint is out of scope for this specification.
+      </t>
+        
+      <section title='Authenticator Association List Request' anchor="list-req">
+        <t>
+          The client makes a request to the authenticator list endpoint
+          including no parameters.
+        </t>
+        <t>
+          The access token MUST be sent as a Bearer Token, per section 2 of
+          <xref target='RFC6750'/>.  It is RECOMMENDED that the access token be
+          sent using the Authorization header field.
+        </t>
+        <figure>
+          <preamble>
+            For example, the client makes the following HTTPS request (line
+            breaks are for display purposes only)::
+          </preamble>
+          <artwork><![CDATA[
+GET /associate HTTP/1.1
```

Comment:
The spec doesn't mandate the name of endpoints, only the processing of requests and associated payloads.  I'd like to keep the spec as close to possible as dynamic registration (including how the endpoints are organized).  That being said, however, its up to the implementation to choose what to name endpoints, and there is no requirement to use the endpoints in the spec's examples.  Either of the alternatives you mention are fine, and I don't have any particular preference.

(Personally, I tend to go with whatever the spec's use in examples, just for consistency.  At the end of the day, its a computer making a programmatic call and the "semantics" of naming don't matter much, and so I prefer not to get hung up on naming issues, because it seems like a waste of time to me :) )